test/driver.spec.js

```diff
@@ -161,6 +161,26 @@ describe('did:key method driver', () => {
       });
     });
 
+    it('should resolve an individual ecdsa multikey did', async () => {
+      const did = 'did:key:zDnaeucDGfhXHoJVqot3p21RuupNJ2fZrs8Lb1GV83VnSo2jR';
+      const mutikeyDid =
+        `${did}#zDnaeucDGfhXHoJVqot3p21RuupNJ2fZrs8Lb1GV83VnSo2jR`;
+      const didKeyDriverMultikey = driver({
+        verificationSuite: createVerificationSuite({
+          generate: EcdsaMultikey.generate, from: EcdsaMultikey.from
+        })
```

Comment:
Does this simpler form not work?

```suggestion
        verificationSuite: createVerificationSuite(EcdsaMultikey)
```

Reply:
It works, updated here [4ca53e6](https://github.com/digitalbazaar/did-method-key/pull/54/commits/4ca53e62026fd3724e4fbc5bde4df76853abc36f)